src/clusterfuzz/_internal/base/tasks/__init__.py

```diff
@@ -368,14 +368,19 @@ def __init__(self,
                eta=None,
                is_command_override=False,
                high_end=False,
-               extra_info=None):
+               extra_info=None,
+               is_from_queue=False):
```

Comment:
is this also for testing only? this seems like a rather confusing flag to add here here. 

Can you also explain a bit why you need this and how it's helpful for testing?

Reply:
It's for testing fuzzing on batch in prod.
Basically, this PR adds a fuzz task scheduler. But we need a mechanism to ensure these fuzz tasks get scheduled on batch instead of just run by regular bots.
This is only needed for that purpose and I will remove it when we have moved all of fuzzing to batch.

Reply:
OK! Please add a tODO here in that case.